### PR TITLE
Chen dataloader

### PR DESCRIPTION
**Pull request by kshannon**
adding in chen's code.

### CODE REVIEW

**Comment by kshannon** [comment]
Hi five. Very nice!

File: src/dataloader.py [hunk 1]
```diff
@@ -7,86 +7,34 @@
 import os
 import pydicom
 import pandas as pd
+import PIL
```
Comment (tonyreina) [inline src/dataloader.py:10]: What's this PIL thing all about?

Comment (kshannon) [reply src/dataloader.py:10]: This is the medicine one needs to take to push past DICOM standards.

File: src/dataloader.py [hunk 2]
```diff
@@ -7,86 +7,34 @@
...
-        # Random number go from 0 to 1. So anything above
-        # self.train_split is in the validation list.
-        trainIdx = idxList[randomIdx < 0.85]
-
-        validateIdx = idxList[randomIdx >= 0.85]
-
-        if testing:
-            self.im_paths = all_paths[validateIdx]
+        all_paths = np.array(sorted(glob.glob(img_path)))
+        if validation:
```
Comment (tonyreina) [inline src/dataloader.py:26]: Get rid of if, just pass validation to single line

Comment (kshannon) [reply src/dataloader.py:26]: Ahh Gotcha.

**Review by tonyreina — APPROVED** [review]

**Pull request merged**